features/creating_order_with_online_payment.feature

```diff
@@ -22,5 +22,6 @@ Feature: Creating order with online payment
         And I select "Paypal" payment method
         And I place this order
         Then I should be notified that order has been successfully created
-        And I should be able to copy payment link for a customer
-        And there should be one not paid nor shipped order for "ned.stark@the-wall.com" in the registry
+        And there should be payment link displayed next to order the payment
```

Comment:
I would say that you should add `a` before `payment`. And what does `next to order the payment` mean?